test/0.8.9/oracle/hash-consensus-access-control.test.js

```diff
@@ -170,18 +172,20 @@ contract('HashConsensus', ([admin, account1, account2, member1, member2, strange
 
     context('setReportProcessor', () => {
       it('should revert without MANAGE_REPORT_PROCESSOR_ROLE role', async () => {
+        const reportProcessor2 = await MockReportProcessor.new(CONSENSUS_VERSION, { from: admin })
         await assert.revertsOZAccessControl(
-          consensus.setReportProcessor(member1, { from: account1 }),
+          consensus.setReportProcessor(reportProcessor2.address, { from: account1 }),
           account1,
           'MANAGE_REPORT_PROCESSOR_ROLE'
         )
       })
 
       it('should allow calling from a possessor of MANAGE_REPORT_PROCESSOR_ROLE role', async () => {
+        const reportProcessor2 = await MockReportProcessor.new(CONSENSUS_VERSION, { from: admin })
```

Comment:
I think it can be moved above into the context and used in both `it` blocks

Reply:
Ok, I will move it